pkg/apk/implementation_test.go

```diff
@@ -48,9 +49,10 @@ QwIDAQAB
 
 var (
 	testPkg = repository.Package{
-		Name:    "alpine-baselayout",
-		Version: "3.2.0-r23",
-		Arch:    testArch,
+		Name:     "alpine-baselayout",
+		Version:  "3.2.0-r23",
+		Arch:     testArch,
+		Checksum: []byte{44, 186, 182, 168, 51, 107, 75, 250, 145, 158, 28, 80, 222, 27, 24, 254, 193, 219, 66, 119},
```

Comment:
Can you comment how you generate this? If we change this or add a new test package I don't know how I'd get this value. Just from the APKINDEX?

Reply:
Great idea.

Reply:
Added some test logging that looks like this:

```
        implementation_test.go:337: did not find cachedPackage("alpine-baselayout") in /var/folders/96/3cdgttts3698hyjg_tydpw5c0000gn/T/TestFetchPackagecache_miss_network_should_fill_cache2563250532/001/https%3A%2F%2Fdl-cdn.alpinelinux.org%2Falpine%2Fv3.16%2Fmain/aarch64/alpine-baselayout-3.2.0-r23: stat /var/folders/96/3cdgttts3698hyjg_tydpw5c0000gn/T/TestFetchPackagecache_miss_network_should_fill_cache2563250532/001/https%3A%2F%2Fdl-cdn.alpinelinux.org%2Falpine%2Fv3.16%2Fmain/aarch64/alpine-baselayout-3.2.0-r23/.ctl.tar.gz: no such file or directory
        implementation_test.go:341:   found "1a3a8e47d2287da6d505d973412cee1ad64bcc17bc5995069e4e932055ecb0c4.dat.tar.gz"
        implementation_test.go:341:   found "2cbab6a8336b4bfa919e1c50de1b18fec1db4277.ctl.tar.gz"
        implementation_test.go:341:   found "2cbab6a8336b4bfa919e1c50de1b18fec1db4277.sig.tar.gz"
```

And a comment that indicates you this is the decoding of `2cbab6a8336b4bfa919e1c50de1b18fec1db4277` from `ctl.tar.gz`.